update_reminder.sh

```diff
@@ -1,10 +1,53 @@
 #!/bin/bash
-MARKT_LAST_CHANGE=$(wget -q -O- https://wo-ist-markt.de/cities/berlin.json | sed -n 's/.*aktualisiert am \(.*\)",/\1/p')
-BERLIN_LAST_CHANGE=$(wget -q -O- https://daten.berlin.de/datensaetze/berliner-wochen-und-tr%C3%B6delm%C3%A4rkte | sed -n '/.*Aktualisiert.*/,$p' | sed -e '/.*end.*/,$d' | sed -n 's/.*<span .*>\(.*\)<\/span.*/\1/p')
-
-if [[ "$MARKT_LAST_CHANGE" != "$BERLIN_LAST_CHANGE" ]]; then
-  echo "Dates don't match!"
-  echo "daten.berlin.de: $BERLIN_LAST_CHANGE"
-  echo "wo-ist-markt.de: $MARKT_LAST_CHANGE"
-  exit 1;
-fi
+
+set -e #exit on error
+
+#get HTTP code from site and exit 1 iff it is not 200
+function failOnNon200HttpStatusCode {
+  SITE_NAME=$1
+  HEADER=$2
+  HTTP_STATUS_CODE=$(echo "$HEADER" | sed -n 's|.*HTTP/... \([0-9]*\).*|\1|p' )
+  if [[ "$HTTP_STATUS_CODE" != "200" ]]; then
+    echo "$SITE_NAME returned a non 200-code" > /dev/stderr
+    echo "$HTTP_STATUS_CODE" > /dev/stderr
+    exit 1
+  fi
+}
+
+function checkGeoJSONAPIExists {
```

Comment:
What is the native way to spell function names in shell scripts? Python uses underbars `check_geojson_api_exists`, Java camelcase  `checkGeoJsonApiExists`, ....

Reply:
I never saw a styleguide for bash, I just used camel case which is at least used often on Stack Overflow :D